dashboard/src/modules/components/AuthComponent/common-components.jsx

```diff
@@ -130,38 +106,6 @@ export const AuthForm = () => {
   );
 };
 
-export const PasswordConstraints = (props) => {
-  const { checkConstraints } = props;
-  const iconList = {
-    indeterminate: <TimesIcon style={{ color: "#D2D2D2" }} />,
-    success: <CheckIcon style={{ color: "green" }} />,
-    error: <CloseIcon style={{ color: "red" }} />,
-  };
-  const passwordLength = useSelector((state) => state.userAuth.passwordLength);
-  return (
-    <>
-      <h4>Passwords must contain at least:</h4>
-      <div className="contraints-container">
-        {passwordConstraintsText.map((constraint, index) => {
-          const variant = checkConstraints[constraint.name];
-          return (
-            <HelperText key={index}>
-              <HelperTextItem
-                name={constraint.name}
-                variant={variant}
-                icon={iconList[variant]}
-              >
-                {constraint.name === "passwordLength" && passwordLength}{" "}
-                {constraint.label}
-              </HelperTextItem>
-            </HelperText>
-          );
-        })}
-      </div>
-    </>
-  );
-};
-
 export const NoLoginComponent = () => {
```

Comment:
I believe that `NoLoginComponent` is dead code, now that `dashboard/src/modules/components/AuthComponent/SignupForm.jsx` is no longer using it.

Reply:
No, `NoLoginComponent` component means if you want to proceed without SSO login.

Reply:
Ah, I overlooked the other reference in `common-components.jsx`.  Nevermind. 🙂